core/scheduler.py

```diff
@@ -77,13 +80,15 @@ def _runners_run_next_jobs(self):
                     self.object_ids[i] = self.runners[i].run_job.remote(self.queue.popleft())
 
     def _start_cron(self, run_frequency_mins):
+        """ CRON job to run next jobs on runners, if applicable. """
         logging.info("Starting cron...")
         schedule.every(run_frequency_mins).minutes.do(self._runners_run_next_jobs)
         schedule.run_pending()
         logging.info("Cron started!")
 
-    #scheduler is a singleton, so need a way "start over"
+    #
     def reset(self):
+        """scheduler is a singleton, so need to get a new instance inplace """
         self.queue = deque()
         self.processed = []
 
```

Comment:
Remove the commented code below this line.

Reply:
Removed.